e2e/test/MessageReceiveFaultInjectionPoolAmqpTests.cs

```diff
@@ -0,0 +1,640 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
See above: name of the file needs to be the name of the class. We can add a .<specifier>.cs after.